bridge/engine/EngineMixerAudio.cpp

```diff
@@ -461,7 +461,8 @@ void EngineMixer::processAudioStreams()
             for (auto& stream : _engineAudioStreams)
             {
                 auto& peerAudioStream = *stream.second;
-                if (peerAudioStream.remoteSsrc.isSet() && peerAudioStream.transport.isConnected() &&
+                if (stream.second->endpointIdHash != audioStream->endpointIdHash &&
```

Comment:
This was a bug slip through. Mix stream is already subtracted because it is contributing, but since it is also a neighbour of itself it is subtracted twice. The analyzeRec did not pick up the frequency because the audio level is different for mix stream.